src/app.rs

```diff
@@ -52,9 +52,9 @@ pub struct Cli {
     #[arg(short = 'R', long, conflicts_with = "tree")]
     pub recursive: bool,
 
-    /// For ls compatibility purposes ONLY, currently set by default
+    /// This will take precedence over any --size=<mode> option
```

Comment:
IMO this can be described as "alias for --size=default (included for compatibility with ls)" or something along those lines, then configure this argument and `--size` to conflict with each other. So instead of choosing precedence when both are used, one must use either one or the other (or none).